test/e2e/systemd_activate_test.go

```diff
@@ -72,7 +72,7 @@ var _ = Describe("Systemd activate", func() {
 		defer os.RemoveAll(tmpRootDir)
 
 		// When SELinux is enabled, a storage root directory should be
-		// labeld with a specific value
+		// labelled with a specific value
```

Comment:
I say, old chap, does this spellchecker suggest `labelled` with double-ell, wot? On this side of the pond, one more traditionally sees the single-el variant, "labeled".

Reply:
And I thought we revolted against England so we would not have to follow their spelling rules.